source/ht_scene.cpp

```diff
@@ -413,5 +445,38 @@ namespace Hatchit {
             }
             m_gameObjects.clear();
         }
+
+        /**
+         * \brief Creates empty GameObject and adds it to the scene.
+         */
+        GameObject* Scene::CreateGameObject()
```

Comment:
This function kinda seems misleading, since it always returns a nullptr.  Should mention that in documentation.